discovery/gossiper.go

```diff
@@ -9,7 +9,7 @@ import (
 	"sync/atomic"
 	"time"
 
-	"github.com/coreos/bbolt"
+	bolt "github.com/coreos/bbolt"
```

Comment:
why this change?

Reply:
Had to add it because it wouldn't compile otherwise. Although it no longer seems to be a problem...